File: src/algorithms/convert-temperature.js
/**
 * Given a non-negative integer rounded to two decimal places that denotes the 
 * temperature in Celsius, converts Celsius into Kelvin and Fahrenheit and 
 * returns the temperatures in an array in the form [Kelvin, Fahrenheit].
 * 
 * @param {number} celsius - An integer parameter
 * @returns {number[]} - Returns Kelvin and Fahrenheit conversions in an array
 */
function convertTemperature (celsius) {
  const fahr = celsius * 1.8 + 32;
  const kelvin = celsius + 273.15
  return [kelvin, fahr];
}

File: src/algorithms/convert-temperature.py
def convert_temperature(celsius):
    """Converts a given Celsius temperature to Kelvin and Fahrenheit.

    You are given a non-negative floating point number rounded to two decimal places celsius, that denotes the temperature in Celsius. You should convert Celsius into Kelvin and Fahrenheit and return it as an array ans = [kelvin, fahrenheit]. Return the array ans.

    Args:
        celsius (Float): A positive float number.

    Returns:
        List[float]: Returns a list with a Celsius temperature converted to Kelvin and Fahrenheit.
    """
    kelvin = celsius + 273.15
    fahrenheit = celsius * 1.8 + 32
    return [kelvin, fahrenheit]
